tests/models/test_vision.py

```diff
@@ -6,33 +6,32 @@
 from pl_bolts.datasets import DummyDataset
 from pl_bolts.models import GPT2, ImageGPT, SemSegment, UNet
 
+pl.seed_everything(42)
```

Comment:
this shall be inside functions

Reply:
Same comment as above. This is defensive programming, to avoid unnessary errors later :smiley:

Reply:
I tihnk globally seeding will avoid duplication of code and possible calls?
We can be determinstic with a seed say 0 or 42 (scikit-learn :smiley: number)